packages/strapi-hook-mongoose/lib/index.js

```diff
@@ -90,6 +91,18 @@ module.exports = function (strapi) {
           return cb(errMsg);
         }
 
+        try {
+          // Require `config/functions/mongoose.js` file to customize connection.
+          require(path.resolve(
+            strapi.config.appPath,
+            'config',
+            'functions',
+            'mongoose.js'
+          ))(instance, strapi.connections[connectionName]);
```

Comment:
I think you want to use `strapi.config.connections` because `strapi.connections[connectionName])`is `undefined`.

Reply:
If you confirm it's the case, please update this file then we will be able to merge this PR!

Thank you!

Reply:
@lauriejim I will change that. 

By the way, i've got the idea from `strapi-hook-bookshelf` so maybe you can look at that file to see if the problem is there too?

Reply:
@lauriejim By looking again, there is the `connection` variable from the `_.forEach` above. Maybe just pass that is clearer, right? 

Waiting for your response

Reply:
Yap we can use `connection` as second param.

And can you also update the the same thing in `strapi-hook-bookshelf` please.

Reply:
Go go go! 🎉